\lhead{\emph{Introduction}} 
\chapter{Introduction}
\begin{table}[!ht]
\begin{tabular}{|l|}
\hline
~\\

\textbf{Résumé~}(see \textit{infra} for English text)\\
\parbox{15cm}{De récents progrès en chimie organique et en biologie moléculaire ont permis la constitution de géantes librairies de molécules, qui sont respectivement des médicaments potentiels et des composés conçus pour aboutir à la sur- ou sous-expression d'un gène d'un organisme donné. Ces produits nécessitent d'être testés : il importe de vérifier la toxicologie comme l'effet attendu des médicaments potentiels. L'existence de librairies de petites molécules comme de siRNAs permettent de tester systématiquement la fonction des gènes d'une espèce. 

D'autre part, des milliers de nouveaux produits chimiques sont synthétisés à des fins industrielles chaque année. Il importe également de les tester, ce qui fait de la toxicologie environnementale un troisième champ d'application majeur pour les cribles biologiques.

Un crible biologique est un ensemble d'expériences conçu pour tester en parallèle l'effet d'un ensemble de composés sur une action biologique spécifique dans un organisme donné. Une expérience de vidéomicroscopie à épifluorescence est une expérience durant laquelle de multiples images d'échantillons fluorescents sont acquises au cours du temps. Cela fournit beaucoup d'informations supplémentaires par rapport à la prise d'une unique image, comme par exemple l'observation d'évènements rares ou encore l'ordonnancement des évènements observés.

Les données de cribles biologiques à haut débit réalisés avec de la vidéomicroscopie sont donc très riches ; leur analyse nécessite la mise au point de techniques statistiques multivariées robustes. La question est donc de savoir comment développer optimalement des méthodes analytiques pour de telles données. Nous présenterons dans le chapitre~\ref{chap:motiw} le premier workflow pour l'étude de la motilité cellulaire individuelle dans de telles données. Le chapitre~\ref{chap:reuse} appliquera ce workflow ainsi que d'autres aux données du projet Mitocheck~\cite{pmid20360735}, démontrant l'utilité de la ré-utilisation de telles données. Enfin, le chapitre~\ref{chap:xbsc} développera une approche méthodologique pour l'analyse de données de vidéomicroscopie à fin de criblage en toxcologie environnementale.}\\
~\\
\hline
\end{tabular}
\end{table}
\clearpage

\paragraph*{Biological screens}~\\
Progress in organic chemistry and molecular biology have led to the constitution of giant libraries, respectively of putative drugs and potential biologically active small molecules, and engineered organisms or proteins for gene silencing or overexpression. As an example, the Biomolecular Screening Facility of the EPFL (Lausanne, Switzerland) has a collection of 65,000 compounds and 130,000 small interfering RNAs (siRNAs\footnote{Small interfering RNAs are short double-stranded RNA molecules which interfers with the expression of genes that present complementary nucleotide sequences.}), while most bio-technological companies offer to ship custom genome-wide siRNA libraries over a fortnight. Pharmaceutical industry libraries are impressive as well, most containing more than one million compounds.

Putative drugs need to be tested for the expected biological effect and against undesirable secondary effects. Understanding their mode-of-action, which screening experiments can help to do, is also a major concern during drug discovery, and often its rate-limiting step~\cite{pmid15547975}. Screens have therefore become a major component of drug discovery processes~\cite{pmid22155864}. On the other hand, the development of biomolecular engineering, and small molecule and siRNA libraries at (more) affordable costs, has led to a significant increase in functional genomic screens, which test for gene function and gene relations. For example, \cite{pmid12140549} exhaustively engineered gene-deletion mutants of the yeast \textit{Saccharomyces cerevisiae}, while \cite{pmid16511445} performed a genome-wide siRNA screen in the fruit fly \textit{Drosophila melanogaster}.

Finally, progress in organic chemistry has also led to the explosion of the number of new molecules which are synthesized each year for industrial purposes (e.g. pesticides, plastics, food additives). This calls for the development of systematic testing experiments, both for the desired action and against undesirable effects on living organisms. Hence a third major field of application for biological screens is Environmental Toxicology.

\textbf{Biological screens} are experiments which are designed for testing a set of compounds for a specific biological action in a given organism. The latter can be any organism in which the action is easily detected, such as a fish (\textit{Danio rerio}), a fly (\textit{Drosophila melanogaster}), a worm (\textit{Caenorhabditis elegans}) or a human cell (\textit{Homo sapiens}). The biological action which is tested can go from a simple univariate assessment of cell death, to the multivariate quantification of an effect on a complex cellular phenotype such as cell division or motility. This will determine the screen \textbf{content}, which would respectively be \textbf{low} and \textbf{high}. Another important parameter of biological screens are their \textbf{throughput}. It ranges from \textbf{low}, for example when testing a dozen of carefully selected compounds, to \textbf{high} for hundreds of thousands of siRNAs in the case of a genome-wide screen. 
\begin{figure*}[ht]
\centerline{\includegraphics[scale=0.28]{figures/various2.png}}
\caption{Images of a wild-type C. elegans worm (top left, nuclear staining,~\cite{plos_elegans}), 12 D. melanogaster larvae (top right, colors: spatial repartition of different mRNAs,~\cite{pmid17923096}) and human breast cancer cells (bottom, red: DNA, green: cytoplasmic membrane, our data)}
\label{image_examples}
\end{figure*}  
\paragraph*{Time-lapse microscopy}~\\
Time-lapse microscopy experiments are experiments in which images are regularly acquired over time. Hence they produce rich 3 or 4-dimensional datasets: they are high-content (HC) almost by definition. Most time-lapse data comes from fluorescent samples. Fluorescent labelling has the advantage of being presently very affordable, (mostly) non-toxic to living organisms and flexible. It makes it possible to easily follow a single protein of interest, which would have been delicate in bright-field images. Furthermore, although it demands either the addition of some reagent when acquiring for a few minutes or hours, or cell genetic modification when acquiring repeatedly for a few days, fluorescence images are easier to segment and analyze than bright field images. This generalized use of fluorescent proteins was permitted by their recent discovery, which started by the green fluorescent protein (GFP)~\cite{pmid8303295},~\cite{pmid19575655}.

The use of time-lapse microscopy rather complicates data acquisition: samples need to be maintained in the appropriate atmosphere at the appropriate temperature, and one should be certain to avoid phototoxicity from repeated light exposure. Nevertheless, they provide a wealth of information which it is not possible to access otherwise. First of all, time-lapse microscopy experiments seem natural for studying dynamic processes such as cell division or cell motility. Furthermore, they enable to visualize very transient events, which are barely observed in endpoints assays, such as early anaphases~\cite{pmid20360735}. Time-lapse microscopy experiments also permit to establish causality links between phenotypes~\cite{pmid15539606}. Indeed, it makes it possible to observe the order in which phenotypes occur, therefore enabling to determine which one leads to the other. It also makes it possible to study cell population heterogeneity in response to gene silencing or chemical exposure: tracking cells over time permits to perform \textit{in silico} cell alignment, and therefore to determine if there exists cell subpopulations with different phenotypic stories. This is not surprising given the stochastic nature of gene expression~\cite{pmid18957198}, and can already been found in the Event Order Maps of~\cite{pmid20360735} (although it was not formalized in that direction in the latter). 

Despite more complex experimental procedures, time-lapse microscopy therefore has a real advantage over endpoint assays when studying complex phenotypes: they truly permit to functionally dig into the consequences of either gene silencing or chemical exposure. As such, it started being used approximately 15 years ago, although some pioneering studies date back from the 1980s (e.g.~\cite{pmid6684600}, see~\cite{pmid19575655} for a review on quantitative time-lapse fluorescence microscopy in single cells). We hereafter refer to time-lapse fluorescence microscopy data by time-lapse data.

\paragraph*{Analysis of high-throughput time-lapse screening data}
~\\ High-throughput (HT) screening experiments have only been made recently possible by the development of screening robots, automated microscopes and measurement devices, and relevant software. These tools are necessary for preparing and performing the experiments. In the case of low-content experiments, result analysis remains simple, although it should not be forgotten that statistics on large datasets should not be done as on small ones (see paragraph on the control of false discovery rate in section~\ref{sec:fdr}). Quality control procedures shall be included in the pipeline as well.

On top of quality control and large dataset statistics, robust and efficient multivariate analytical methods are necessary to deal with HC HT screening experiments. This applies to HT time-lapse screening data as well. It is indeed a specific type of HC HT screening data, in which one data dimension is time. Most of the time, analyzing such datasets demands to combine computer vision methods to multivariate statistical algorithms for significant effect detection. These methods should be tailored to the biological process which is studied, but they all have in common the challenges to be robust when faced with noise, and scalable as dataset sizes increase. The question is to know how to develop analytical methods for optimally exploiting such datasets.

A first biological process which seems natural to be studied with time-lapse microscopy is single cell motility. A systematic functional genomics approach to cell motility is all the more needed since all the involved proteins and pathways are not yet known. Nevertheless, it was never studied using multivariate statistical tests in HT settings. During this thesis, we therefore developed a generic methodological workflow for studying single cell motility in HT time-lapse screening data, which will be presented in chapter 2. As will be detailed, an \textit{ad-hoc} statistical procedure indeed had to be developed.

The generic quality of this workflow enabled it to be applied to an existing dataset, the Mitocheck dataset (see section~\ref{sec:mito_prez} for a presentation of this dataset). This revealed the quantity of unexploited information in this dataset, and more generally the wealth of existing HT time-lapse screening data which can be re-used to different purposes than the original experimental design. Proofs of this constitutes chapter 3, which presents how an ontology of single cell trajectories could be extracted from the Mitocheck dataset (section~\ref{sec:ontology}). The latter was also used for detecting cell cycle genes (section~\ref{sec:cellcycle}). In section~\ref{target_inference}, it also permitted to perform drug target inference on an unpublished time-lapse drug screen, which made it necessary to develop a new distance.

Finally, as was mentioned in the beginning of this introduction, one of the most important applications of screening is in Toxicology. Moreover, HT time-lapse screening experiments have never been performed in Environmental Toxicology. We therefore developed a robust methodological workflow and its visualization Web-interface, for conducting and analyzing HT time-lapse screening data in Environmental Toxicology. This composes chapter 4.

Before diving into the main matters, all the datasets which were analyzed in the course of this thesis, as well as the software which we used to this end, are briefly described in the following section.
%What is the topic and why is it important? State the problem(s) as simply as you can. How does it fit into the broader world of your discipline? 
%Where did the problem come from? What is already known about this problem? What other methods have been tried to solve it? 
%mentionner la toxicologie environnementale. Ce qu'on fait c'est de la mise en place d'outils. Etre un peu global sur le time-lapse en criblage. Il y a toxicology(envtal pharmaco)/genetic. Explain what a screen is, give examples
%
%What are the challenges and questions that come up in such a setting => pour que ca nous arrange
%
%Not necessarily very long. Just prepare the reader to know what screens are, their scope. Also give a reasoning fil rouge for the rest, for it to be clear for the reader
\section{Data sets}
Four time-lapse datasets were used in our work, which will briefly be presented in this section: the Mitocheck dataset, which is the first genome-wide siRNA time-lapse screen, a PCNA dataset for the study of cell cycle phases, an unpublished drug screen in similar settings to those of the Mitocheck dataset, and an unpublished xenobiotic screen.

\subsection{Mitocheck data set}
\label{sec:mito_prez}
The main dataset which we used is a previously published genome-wide data set of time-resolved
records of cellular phenotype responses to gene silencing, which
were generated for virtually all protein-coding genes~\cite{pmid20360735}. It is publicly available at
\href{http://www.mitocheck.org}{mitocheck.org}. 

For this, arrays of transfection cocktails containing small interfering RNA (siRNA) were spotted directly into live cell-imaging chambers in a 384 format. HeLa cells (ATCC\up{\textregistered} CCL-2\texttrademark) stably expressing the core histone 2B tagged with GFP were seeded on top of the arrays, and imaged 18 h after the transfection for 48 h with a time-lapse of 30 min (Plan10x, NA 0.4; Olympus - see fig.~\ref{mitocheck} for an example). Imaging chambers were sealed during imaging. Each microarray contained 8 negative controls (scrambled: not targeting any gene) and 12 positive controls showing different phenotypes. 
22,612 protein-coding genes have been targeted by at least 2 siRNAs each, in total 51,767 siRNAs. For each siRNA, there is data from at least 3 technical replicates, which created 182,191 quality controlled time-lapse experiments in total. Due to updates in the genome annotation, some reagents could not be mapped to the current ENSEMBL version. In total, the data set contains data for 17,816 protein-coding genes in 144,909 quality controlled time-lapse
experiments.  

HeLa cells are epithelial cancer cells which were derived from the adenocarcinoma of Henrietta Lacks in 1951. Being the first human cell line to survive \textit{ex vivo} for more than a few days, they are very appreciated from cell biologists as they are easy to grow and transfect. Indeed, this cell line is mentioned in approximately 0.3\% of PubMed abstracts although 64\% of its genome has a copy number greater than three~\cite{pmid23925245}. It is not motile as can be the case of other cell lines which are widely used in migration studies such as the epithelial metastatic breast-cancer derived MDA-MB-231 cell line (ATCC\up{\textregistered} HTB-26\texttrademark), or the epithelial metastatic lung-cancer derived NCI-H1299 (ATCC\up{\textregistered} CRL-5803\texttrademark). Indeed, it was the 16\up{th} slowest out of 54 in the first World Cell Race~\cite{pmid22974990}. Gene silencing in this background therefore makes it easier to identify migration suppressors, that is, genes whose silencing will enhance cell motility, rather than migration enhancers, given that HeLa basal cell motility is rather low.
\begin{figure*}[ht]
\centerline{\includegraphics[scale=0.4]{figures/mitocheck.png}}
\caption{Image from a control video of the Mitocheck dataset (white: histone 2B)}
\label{mitocheck}
\end{figure*}  
\subsection{PCNA data set}
In section~\ref{sec:cellcycle}, another published dataset is mentioned, which is related to the study of cell cycle phases. It was published with~\cite{cellcognition} and is publicly available with annotations\footnote{\href{http://www.cellcognition.org/downloads/data}{http://www.cellcognition.org/downloads/data}}.

Briefly, HeLa cells were stably transfected for a red fluorescent chromatin protein (histone 2B fused to mCherry, H2B-mCherry) and a green fluorescent DNA replication factory (proliferating nuclear antigen fused to GFP, PCNA-mEGFP). Cells were seeded on LabTek chambered coverslips for live microscopy, and imaged for 48h with a time-lapse of 6 min (Plan10x, NA 0.5; Nikon - see fig.~\ref{PCNA} for an example). Cells were maintained at $37^\circ$C in humidified atmosphere of 5\% CO2 during imaging. Following this, cell nuclei were segmented using local adaptative thresholding, improved by a split-and-merge approach as described, and samples for the different cell cycle phases were manually annotated using the open-source software Cell Cognition~\cite{cellcognition}.
%
%Automated microscopy with reflection-based laser auto focus was performed on a Molecular Devices ImageXpressMicro screening microscope equipped with a 10× 0.5 numerical aperture (NA) and 20× 0.8 NA S Fluor dry objectives (Nikon) and recorded as two-dimensional time series. The microscope was controlled by in-house–developed Metamorph macros (PlateScan software package, available at http://www.bc.biol.ethz.ch/people/groups/gerlichd). Cells were maintained in a microscope stage incubator at 37 °C in humidified atmosphere of 5% CO2 throughout the entire experiment. 
\begin{figure*}[ht]
\centerline{\includegraphics[scale=0.4]{figures/PCNA.png}}
\caption{Image from a control video of the PCNA dataset (white: histone 2B, green: PCNA)}
\label{PCNA}
\end{figure*}  
\subsection{Drug screen}
In section~\ref{target_inference}, we analyze an unpublished time-lapse drug screen\footnote{Manuscript in preparation}. In this dataset, 25 drugs were screened for their effect on HeLa cells in similar experimental settings to that of the Mitocheck dataset.

Experiments were not conducted in the context of this PhD. They were performed at the Advanced Light Microscopy facility of the EMBL (Heidelberg, Germany) by Beate Neumann, Jutta Bulkescher and Thomas Walter. Briefly, HeLa cells were stably transfected for a green fluorescent chromatin protein (H2B-GFP). Cells were seeded on 384-well plates for live microscopy **h prior to imaging. Drug exposure occurred ***h prior to imaging. Finally, cells were imaged for 48h with a time-lapse of 30 min (Plan10x, ** NA; [MICROSCOPE BRAND]). Cells were maintained at $37^\circ$C in humidified atmosphere of 5\% CO2 during imaging.

\subsection{Xenobiotic screen}
In chapter 4, we analyze an unpublished time-lapse xenobiotic screen, for which we performed the experiments. In this dataset, 5 xenobiotics were screened for their effect on MCF-7 cells.

Briefly, MCF-7 cells (ATCC\up{\textregistered} Catalog N\up{o}HTB-22\texttrademark) were stably transfected for a red fluorescent chromatin protein (H2B-mCherry) and a green fluorescent membrane protein (myrPalm fused to GFP, myrPalm-GFP). Cells were seeded on 96-well plates 24h prior to exposure, and they were exposed to xenobiotics 24h prior to imaging. Finally, cells were imaged for 48h with a time-lapse of 15 min (Plan10x, 0.3 M27; Zeiss - see bottom of fig.~\ref{image_examples} for an example). Imaging plates were sealed during imaging. Our experimental procedures will be detailed in section~\ref{protocoles}.

\section{Software}
We use CellCognition \cite{cellcognition}\footnote{\href{http://cellcognition.org}{http://cellcognition.org}} for segmentation and object feature extraction in all projects. To store, manage and access
screening data, we use a previously published data format CellH5~\cite{Sommer2013}. All scripts are written in the programming language Python 2.7\footnote{\href{http://www.python.org}{http://www.python.org}}
using scipy~\cite{scipy}, numpy, scikit-learn, fastcluster~\cite{fastcluster}, rpy2 and statsmodels, and all plots were generated by matplotlib~\cite{matplotlib}. The Web-based user interface which was used for data visualization and sharing is based on Django\footnote{\href{https://www.djangoproject.com/}{https://www.djangoproject.com/}}, Linux-Apache web-server, mod\_wsgi and SQLite\footnote{\href{https://sqlite.org/}{https://sqlite.org/}}.
The R statistical function stats.p\_adjust was used for adjusting p-values according to the Benjamini-Hochberg procedure~\cite{Benjamini1}. Finally, CPlex\footnote{\href{http://www-01.ibm.com/software/commerce/optimization/cplex-optimizer/}{http://www-01.ibm.com/software/commerce/optimization/cplex-optimizer/}} was used for optimization in the tracking procedure.